package.json

```diff
@@ -21,12 +21,8 @@
     },
     "bundlesize": [
         {
-            "path": "./dist/frontend.article.*.js",
-            "maxSize": "50 kB"
-        },
-        {
-            "path": "./dist/vendor.*.js",
-            "maxSize": "70 kB"
+            "path": "./dist/ga.*.js",
```

Comment:
This is a placeholder. Unfortunately, bundlesize does not appear to support a budget for a group of files here, and I'm not sure it's that useful to apply individual budgets to each.